### PR TITLE
new homework day 3 and 4

### PR DESCRIPTION
**Pull request by dmlatesting**

### CODE REVIEW

File: Day-3-Exercises/main.js [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+What is the date today?
+What is the day today as a number?
+What is the hours now?
+What is the minutes now?
+Find out the numbers of seconds elapsed from January 1, 1970 to now. */
+const myDateObj = new Date()
+console.log('The year:', myDateObj.getFullYear())
+console.log('The month:', myDateObj.getMonth())
+console.log('The date:', myDateObj.getDate())
+console.log('The day:', myDateObj.getDay)
```
Comment (jkalandarov) [inline Day-3-Exercises/main.js]: You should call `.getDay` method by opening parentheses.

File: Day-3-Exercises/main.js [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+//Writ a script that prompt a user to enter hours and rate per hour. Calculate pay of the person?
+let numHours = prompt('Enter number of working hours per week:')
+let rate = prompt('Enter rate:')
+console.log('Your weekly earning is', numHours * rate)
+
+//If the length of your name is greater than 7 say, your name is long else say your name is short.
+let name = prompt ('Enter your name:')
+name.length > 7
+? console.log(`Your name ${name} is long`)
+: console.log(`Your name ${name} is short`)
```
Comment (jkalandarov) [inline Day-3-Exercises/main.js]: Can you refactor this piece of code?

File: Day-3-Exercises/main.js (hunk 1 above)
Comment (dmlatesting) [reply Day-3-Exercises/main.js]: fixed

File: Day-3-Exercises/main.js (hunk 2 above)
Comment (dmlatesting) [reply Day-3-Exercises/main.js]: Can you give a bit more details on what is wrong with this code?

**Review by jkalandarov — APPROVED** [review]

File: Day-3-Exercises/main.js (hunk 2 above)
Comment (dmlatesting) [reply Day-3-Exercises/main.js]: @jkalandarov Updated

**Review by jkalandarov — APPROVED** [review]

**Pull request merged**